src/legacy/ui/public/agg_types/controls/test_utils.ts

```diff
@@ -19,7 +19,7 @@
 
 import { AggConfig, VisState } from 'ui/vis';
 import { EditorConfig } from 'ui/vis/editors/config/types';
-import { SubAggParamsProp } from 'ui/vis/editors/default/components/default_editor_agg_params';
+import { SubAggParamsProp } from 'ui/vis/editors/default/components/agg_params';
```

Comment:
Could we also export it in `default/index.ts` ?
and then: 
```suggestion
import { SubAggParamsProp } from 'ui/vis/editors/default';
```